src/pages/result/Result.jsx

```diff
@@ -1,16 +1,45 @@
 import React from "react";
 import { useCategory } from "../../context/category-context";
 import "./result.css";
+import { Link } from "react-router-dom";
+import { questions } from "../../db/questions";
+
```

Comment:
- all things are good to merge
- just see why deploy is failed
- code is clean and well formatted